jwql/tests/test_cosmic_ray_monitor.py

```diff
@@ -0,0 +1,175 @@
+#! /usr/bin/env python
```

Comment:
I made some tweaks to get these tests working, and now most of them pass, however a few do not:

```
============================================================= short test summary info ==============================================================
FAILED jwql/tests/test_cosmic_ray_monitor.py::test_magnitude - assert -0.15904 == -2.77504
FAILED jwql/tests/test_cosmic_ray_monitor.py::test_get_cr_mags - assert [-0.15904] == [-2.77504]
```

Do you know what is going on here?